opentracing-spring-cloud/src/main/resources/META-INF/spring.factories

```diff
@@ -1,10 +1,11 @@
 org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
-io.opentracing.contrib.spring.cloud.feign.FeignTracingAutoConfiguration,\
```

Comment:
please add only what has changed

Reply:
the space before each AutoConfiuration was needed and I have seen in all spring.factories hence thought to update it..

Reply:
since I saw in all the spring.factories - having space before each autoconfig line thought to add it for readability.. removed in new commit